naga/src/front/atomic_upgrade.rs

```diff
@@ -212,3 +212,82 @@ impl Module {
         Ok(())
     }
 }
+
+#[cfg(test)]
+mod test {
+    fn atomic_test(bytes: &[u8]) {
+        let _ = env_logger::builder().is_test(true).try_init();
```

Comment:
Wouldn't it make more sense to `.expect` this `Result` than ignore it?

Reply:
I would say yes if this were `main`, where one would expect that logging would only be set up once. But in a test it may be that the logger has already been initialized by some other test, in which case I don't want to panic. 

Usually when using logging in a test I'll use `ctor` to ensure that we only init once, but that would turn on logging for all tests and introduce a new dependency.

Reply:
Oh, right. I forgot about all that mess.